.evergreen/functions.yml

```diff
@@ -694,7 +694,7 @@ functions:
 
           #if [[ "$IS_UBUNTU" == "true" ]]; then
             # TODO: linux_deb
-            # TODO: linux_tar
+            npm run --unsafe-perm --workspace @mongodb-js/compass-smoke-tests start -- --package=linux_tar --tests=time-to-first-query
```

Comment:
In this case it should be safe to run autoupdate-from because it won't auto-update, but it will tell the user that there is an update. And there's test code for that.

But we can also enable that in a separate PR.